GEOSlandassim_GridComp/clsm_ensupd_read_obs.F90

```diff
@@ -4497,6 +4522,17 @@ subroutine read_obs_SMOS( date_time, N_catd, this_obs_param,      &
     
     found_obs = .false.
 
+    ! determine operating time range of sensor
+
+    date_time_obs_beg = date_time_type(2010, 5,24, 0, 0, 0,-9999,-9999)
```

Comment:
@gmao-qliu:  What is the earliest date for which we have good-quality SMOS obs?  My memory suggests that useful SMOS science data only start in July 2010.

Reply:
We have continuous valid data starting from 2010/05/24, so the date seems correct. We used 2010/06/01 onward for estimating the obs scaling parameters for SMAP in the past.